Kochbuch/src/app/rezeptansicht/rezeptansicht.component.html

```diff
@@ -1,91 +1,58 @@
MaVZ3yOdLCWN7jPlugKk10MYzFWVcW2H8yI7XU5BFIDhruy1eO8kg1V2+XosYwre/0rT0e/EEgsb0gRyD5Ce3tXYzR/2pYHdxcR9fevPPGFsyWyzJkPE3btTvcTVil4u0NrSczRjMbHIrlHB7dRXp+j3set6UsFzjzNvfvXG+INJlsLh12nZ2NZyiUmYlrcGNtrcqa1LKMSXcXdCwzWHNG27IOCK09FuvIuUMn3AaybsM9DmtWuERVmZFHRV6Vg3/haGWRnkOS3JJHWtzTboSBWQgrV+5ZcZOK4qlKM1dkHmEvgeyju/N3SeX1KZ4JrN8ca8fC9vaW2lGOOaQEsMZwK7nxBqEdlayzP0QZrwHWFv/EGozXgR23NhV54FefKMI3SGzvPh5r+oavc3r387SRgAKDwAa6O+06wnX95bxH14rmvCNgNG0sRscTSfM/1rSudRjiGGfr0rllVQzREkcEaxxgKijAA7Vwviy4up74PKuIkGEKelbM9675CCqEyecD58gx6U6cK09Ix0JZlaHrF6LgRJKTAPv7ufyrek1B5M7AxrOEdjaYbbgE9+5q/Cst2m5UMcfY4xmta2Dl8VSVvJAipd3s44wR9DUNheFZczNtyeCa0pLLAwBXPazB+8Vd2FXlqyoYZ05+0NKdTkfMjptSSBpre6RoTMnDAkfMKz9SK3TqY1YsvTb0rlNGC3F9I3ms6BsAk12D7bexcqdrY+XPevT5VUWp6EZxavY5bW7qSOWPcW2oecHBFOvNQhkR2gaV0/wBs+1YmsTSzTnzHyvfHrVUXLCArCv1FWoqKD6zG6Rlat++uSV4FSaNpzyS7myqevTNS6fC01yZZQwIPC4rf8ouu1Yj+WKyqSsrJnLWrczdi7BBFEqgFSwGOamltYtodgCQOGrMkhaOLdMdiLzWdJcXuqP8AY9Njdl6cf1NccYe9c5PM9Q8O/Eq2t4IrLUJzJJGQgZeSR7+9ei2+v215FC0bB1Ydufz9K8L0XwRDZSxzX6+fckZ4+6hrXuJZ9PYmzmaM5ycHg11LHxg1F6mkavRnonjLw3B4htNsO1Lkcxv059DXCaZoGraVqX2a/wBqrsJDAnB/GqkHxBuYVSO4BaSNuGQ4z9atXXxatruNbe+s2WToJV7VtL2VVXW50U6lndMyr820GrLGYw5J2mQ85NbUH2WCIvNiFAeR0JrnvEHktcJcWUiShufl7E96qkT3DBrhyx/SuatWhFK61NpV/Z6RNjUNdlnYx2mY4hwKrwo5G+ZizepNFpaheWFJeyhFIHH9Kyg5S96RxSnKbKOqXQjQt6cKPU1zoVpHLNksTmrlyTczE5+ReBUsVvnoK0cuh0wjyoghi5ra06yMrDcMKOtOsbHcAWHH866KytdiAYGa68PhnJ80tiKlS2iEs7UIM4AJrSii7iiGLPSr0EHNeqo20RztktjGc4xW/axkAcVn2kWCOMVs2y4AyOKtEstQDj0qYnPSmKOOOlTxR7jxVCHQrkjAzWnCuBUMKADFT/dGaAJS+0U1eTmo1JPJqVBgZNIB4O0VzXiPU1DC3RupwcVo61qC2lu3PzEV57Jctc36u2cbqBlzVB/on0YVkYya2dU/49PqwrHAwaBiL/rBXSWX/HvH9K5sj51rotLcPaL6jikgKOpri6J9QDVZRzV/WE+eNvbFUlFICRRz9akVefwNEa8ip1TmnYBbcfJirUa1FCuOPeriLTEOgXtXTWo8ywA9UxXOxLzXSaUc2oHocUwOrfoKmtuhqBqntvumkMLntUIqW57VFQDFopKWgApe1Jnil7UAWIv9XVdh1qzF/q6rnqaSALVvKnDdjw1YnjXTg0EhA+SUH8DW0eRU91AL/THiP3gOD70no7j3VjxjQ7l4RJEjFZoH6eort0kg1yw2SACcDvXC6xE2k+IUmYYilO1vrWzG72s6TQng8/UVZJi6vpD2szBh34rOVduQa9PkSDV7TDgCTHWuI1TSpLWYgg7fWuapB9B3NTw3ISvONtbN/cqkZ3nAriIrmW0B8ptprB8R6rqd0cGUhB/driqqSVooaVzpNb1O0lUpIUZR1Ga5S61i2iBECDHsMVgM02Dkkt796y9cupobNpY4tzDooHeuVYVS1mNoPEniy+tlcW0Semc8isrw34onvbkw3pLPnIOOlcjJ9r1G8fIYO5+b2rsdC0cWsYIX5j1J71pCjCL91COqkvMoMGmRySTMEQEsemKWwsJbqZYol3Mf0r0HQ/DsVlGHZd0p6k1VWtyaLcluxi6VoRaBTegNg7guOhrZNoAMADFbRhC+wqjeSrEpArmer5p7kmJfqsUZPGa5S6Id5WIQqwwQTWvr1+qowByx6AVh6P4Yu9euGkune1tfrgtWf1lKXKPpqc75y216BbLlOwX1rSup7m6gO5GGOFA/rXXx+GNP0kbYF3v3Zjk1G8McayeWo5GCKUMWlKz0Roq7irRPONS0mWKAPNNsycsfatO00eTTbYSyqZInAcNjqK17rwp/aOky3aXLmSE5aPParGt+J4bLwvbxXcG66jGEwOCPeu6LhOO9jNyctWZJe3RdzRoqtxuAp627BXMbo4AyD0rndDvHvi8typEW7OOwqbUrm7nYQWCkQ9yOtYNJL3mFzT0yzbV5pI5GCRgYPrXX6ZptrptusVpGqKOrY5Ncf4aW4trn98p6V2KzHYCT+FcNa792OxL1HXUgVTXHeILoOxgil2yHnHrXR6lOkVq8kjYrkrWCOW+Mrt5pILD2FTRoe97w0rGFc2YDfv2IY96m03w+bm6D4Yx+4rptL059Wmllij/dRnncK6u1tYYYgEUVtWqcmkQuc3DpCwIAB+FWI7ML1rdkjA5NUbt1jQk4rnh7zuwuzNu5FgQ1zd5O07lE79TVrU7oyuQppljaFjkiuq7Wh00oW1ZDbWhIAxWva6dkYPX+VaNlp+TtA4H3jWvHZ7MLivSw2EXxTHOp0Rn29r5aqOuK07eHgVYhtc9RV6C2x
```

Comment:
`div` ist ein allgemeines Blockelement. Da das hier mit `d-inline` zum Inlineelement gemacht wird, kann man auch gleich `span` verwenden. 

Außerdem ist das Rating nicht in Sternen, sondern entweder eine positive oder negative Zahl (Upvotes - Downvotes)